src/index.mustache

```diff
@@ -60,7 +60,7 @@
         <p>
             Fumos are generally difficult to purchase, even in Japan, due to their relative scarcity.
             Fumos are out-of-stock as a rule, and Gift typically only manufactures them twice a year, when
-            they open for limited pre-orders. Gift does not ship outside of Japan.
+            they open for limited pre-orders. Gift only ships outside japan through <a href="https://www.amiami.com/eng/search/list/?s_keywords=Ba_00102">AmiAmi</a> store.
```

Comment:
To me this implies that every restock has an AmiAmi sale, which hasn't historically been the case. Maybe something like "For most restocks, Gift does not ship outside of Japan; when they do, the sales go through the AmiAmi store." would be clearer?

Reply:
Lgfm